src/components/tl/ucp/tl_ucp_ep_hash.h

```diff
@@ -23,16 +23,15 @@ static inline uint32_t tl_ucp_ctx_id_hash_fn_impl(uint32_t h, uint32_t k)
 static inline khint32_t tl_ucp_ctx_id_hash_fn(ucc_context_id_t k)
 {
     uint32_t h = 0;
-    h = tl_ucp_ctx_id_hash_fn_impl(h, k.host_id);
-    h = tl_ucp_ctx_id_hash_fn_impl(h, k.pid);
+    h          = tl_ucp_ctx_id_hash_fn_impl(h, k.pi.host_hash);
```

Comment:
Alignment

Reply:
that is what "Clang" produced - fixed manually

Reply:
yeah, maybe because there is no blank line between "h" declaration and assignment